caffe2/CMakeLists.txt

```diff
@@ -975,6 +975,7 @@ elseif(USE_CUDA)
   torch_compile_options(torch_cuda)  # see cmake/public/utils.cmake
   target_compile_options_if_supported(torch_cuda "-Wno-deprecated-copy")  # see cmake/public/utils.cmake
   target_compile_definitions(torch_cuda PRIVATE USE_CUDA)
+  target_link_libraries(torch_cuda PRIVATE fmt::fmt-header-only)
```

Comment:
We do use fmt::format all over ... is this intended? Just checking.

Reply:
Yeah -- it's intended, wanted to use fmt in torch_cuda